core/src/miner/miner.rs

```diff
@@ -486,17 +264,22 @@ impl MinerService for Miner {
         self.mem_pool.write().set_limit(limit)
     }
 
-    fn chain_new_blocks<C>(&self, chain: &C, _imported: &[BlockHash], _invalid: &[BlockHash], _enacted: &[BlockHash])
+    fn chain_new_blocks<C>(&self, chain: &C, _imported: &[BlockHash], _invalid: &[BlockHash], enacted: &[BlockHash])
     where
         C: AccountData + BlockChainTrait + BlockProducer + EngineInfo + ImportBlock, {
         ctrace!(MINER, "chain_new_blocks");
 
         {
-            let fetch_account = fetch_account_creator(chain);
-            let current_block_number = chain.chain_info().best_block_number;
-            let current_timestamp = chain.chain_info().best_block_timestamp;
             let mut mem_pool = self.mem_pool.write();
-            mem_pool.remove_old(&fetch_account, current_block_number, current_timestamp);
+            enacted
```

Comment:
@sgkim126 Could you check this, too?
This code removes enacted transactions from the mempool.
I'm worried if this is wrong, or too inefficient.

Reply:
It seems okay to me.
I can't think of a more efficient way now and, at least, the implementation is not wrong.